src/object/object_print.c

```diff
@@ -700,6 +700,18 @@ help_print_info (const char *command, FILE * fpp)
     {
       logtb_dump_trantable (fpp);
     }
+  else if (MATCH_TOKEN (buffer, "ndv"))
```

Comment:
Any plans to add messages about ndv to `CSQL_HELP_INFOCMD_TEXT`?
```
csql> ;info

ERROR: Info {schema|trigger|deferred|workspace|lock|stats}
        schema [class-name]    - schema information
        trigger [trigger-name] - trigger information
        deferred               - deferred trigger information
        workspace              - workspace information
        lock                   - lock information (client-server mode)
        stats class-name       - class statistics information
        trantable              - transaction information
        logstat                - log information
        csstat                 - critical section information
        plan                   - plan cache information
        qcache                 - query cache information
```

Reply:
no. It is an intermediate stage of development related to statistic improvement. it can be removal later.